packages/nimble-components/src/rich-text/components/mention-view/mention-users-view/tests/rich-text-mention-users-view.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { html } from '@microsoft/fast-element';
+import { RichTextMentionUsersView, richTextMentionUsersViewTag } from '..';
+import { type Fixture, fixture } from '../../../../../utilities/tests/fixture';
+import { waitForUpdatesAsync } from '../../../../../testing/async-helpers';
+
+async function setup(): Promise<Fixture<RichTextMentionUsersView>> {
+    return fixture<RichTextMentionUsersView>(
+        html`<${richTextMentionUsersViewTag}
+            mention-href="users:1"
+            mention-label="John Doe"
+            disable-editing
+            >@John Doe</${richTextMentionUsersViewTag}
+        >`
+    );
+}
+
+describe('RichTextMentionUsersView', () => {
+    let element: RichTextMentionUsersView;
+    let connect: () => Promise<void>;
+    let disconnect: () => Promise<void>;
+
+    beforeEach(async () => {
+        ({ element, connect, disconnect } = await setup());
+    });
+
+    afterEach(async () => {
+        await disconnect();
+    });
+
+    it('should export its tag', () => {
+        expect(richTextMentionUsersViewTag).toBe(
+            'nimble-rich-text-mention-users-view'
+        );
+    });
+
+    it('can construct an element instance', () => {
+        expect(
+            document.createElement(richTextMentionUsersViewTag)
+        ).toBeInstanceOf(RichTextMentionUsersView);
+    });
+
+    it('should set the `mention-label` attribute to the internal control text content', async () => {
+        await connect();
+        expect(element.shadowRoot?.firstElementChild?.textContent).toBe(
```

Comment:
Could you move this into a new page object class with just a single method like `getTextContent(): string`?

Reply:
Moved it to a page object file